public/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+apigility-ui/
+zf-apigility/
```

Comment:
Is this file intentionally here or just mistake? (Isn't it autogenerated by `zf-asset-manager`?)